src/helper.js

```diff
@@ -1,10 +1,29 @@
 var path = require("path");
+var os = require("os");
 
 module.exports = {
 	HOME: (process.env.HOME || process.env.USERPROFILE) + "/.lounge",
-	getConfig: getConfig
+	getConfig: getConfig,
+	expandHome: expandHome,
 };
 
 function getConfig() {
 	return require(path.resolve(this.HOME) + "/config");
 }
+
+function expandHome(path) {
+	var home;
+
+	if (os.homedir) {
+		home = os.homedir();
+	}
+
+	if (!home) {
+		home = process.env.HOME || "";
+	}
+
+	home = home.replace("$", "$$$$");
```

Comment:
Because `$$$$` escapes into `$$`, which is required for the next replace to keep the final `$`.

See https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Global_Objects/String/replace#Specifying_a_string_as_a_parameter